CHANGELOG-4.0.md

```diff
@@ -6,14 +6,16 @@ in 4.0 versions.
 
 * 4.0.0 (xxxx-xx-xx)
 
- * Add `ruflin/Elastica` 3.x support.
```

Comment:
@dbu I don't think it is. But there doesn't seem much damand. So I don't think it matters now.